sign/pending_requests_test.go

```diff
@@ -174,7 +174,10 @@ func (s PendingRequestsSuite) TestConcurrentComplete() {
 		}()
 	}
 
-	s.pendingRequests.Wait(req.ID, 10*time.Second)
+	rst := s.pendingRequests.Wait(req.ID, 10*time.Second)
+	if rst.Error != nil {
```

Comment:
Why not `s.Require().NoError(rst.Error)`?

Reply:
No reason, fixed. Thanks for the heads up.